Cargo.toml

```
@@ -23,34 +23,34 @@ version = "0.0.11"
 actix-cors = "0.7"
 actix-server = "2.5"
 actix-web = "4.9"
-anyhow = "1.0"
+anyhow = "1.0.91"
```

I guess we should follow the pattern of just using version until minor, by not fixing the patch we will have updates automatically in the lock.

```suggestion
anyhow = "1.0"
```

---

fixed